mentoring/public/js/mentoring_assessment_view.js

```diff
@@ -28,7 +28,10 @@ function MentoringAssessmentView(runtime, element, mentoring) {
         return attempts_data.num_attempts >= attempts_data.max_attempts;
     }
 
-    function renderGrade() {
+    function renderGrade(event) {
+        if (event) {
+            event.preventDefault()
+        }
```

Comment:
@Kelketek this function is used both as event handler and normal method - it works just fine, but in my opinion handling event and doing stuff are different responsibilities, and now this function mixes them. So, it might make sense to keep it normal function and wrap it in lambda functions where it's used as an event handler